File: c/1216.c
//1216	Getline One	Accepted	C	0.020	14/03/2017 18:58:36
#include <stdio.h>
#include <stdlib.h>

int main()
{
    char nome[80];
    double total=0, k=0;
    int dist;

    while((scanf("%s",nome)) != EOF)
    {
        scanf("%d", &dist);
        total+=dist;
        k++;
    }
    
    printf("%.1lf\n", total/k);
    return 0;
}

File: c#/1216.cs
//1216	Getline One	Accepted	C#	0.236	16/11/2019 18:14:06
using System; 
 
class URI {
 
    static void Main(string[] args) { 
        string name;
        double total = 0, k=0;
        int dist;
        
        while((name = Console.ReadLine()) != null && name != ""){
            dist = Int32.Parse( System.Console.ReadLine().Trim());
            
            total += dist;
            k++;
        }
 
        Console.Write("{0:0.0}\n", total/k);
    }
     
}
